src/math-translator.js

```diff
@@ -314,6 +321,14 @@ function normalizeTranslatedMath(math, lang) {
         }
     });
 
+    // Remove whitespace in coordinates/intervals
+    // Applied to all langs because all langs can be affected by
+    // tranlatedCoordinates/translateIntervals/translateCoordinatesOrIntervals
+    const orderedPair = getOrderedPairRegexString(lang);
+    const coordsAndIntervals =
```

Comment:
Any value in moving this into `getCoordsAndIntervalsRegex` function as well?

Reply:
This particular regex, which matches all possible combinations, is only used here, so I'd keep as is.